_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -2730,9 +2736,11 @@ mfxStatus VAAPIEncoder::Execute(
         }
     }
 
+    bool isBrcSlidingWindowFrameToleranceMode = (MFX_HW_APL == m_hwType || MFX_HW_CFL == m_hwType);
```

Comment:
That's much better than before! However, this should not be a variable you hold on stack and calculate each time. I wrote this before, here is explicitly once more time:
1. Add 'UCHAR m_BrcSlidingWindowFrameToleranceMode' to https://github.com/Intel-Media-SDK/MediaSDK/blob/5d9da414de71a2c38374ed932295b6786a3c5863/_studio/mfx_lib/shared/include/mfx_h264_encode_struct_vaapi.h#L305 (ENCODE_CAPS structure)
2. Calculate it once in the https://github.com/Intel-Media-SDK/MediaSDK/blob/5d9da414de71a2c38374ed932295b6786a3c5863/_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp#L1332 (VAAPIEncoder::CreateAuxilliaryDevice)